src/conditionals/programs_with_decision_points/hard/q14/MainTest.java

```diff
@@ -0,0 +1,73 @@
+package conditionals.programs_with_decision_points.hard.q14;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+    
+    @Override
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "apple\nbanana";
+        
+        return new Clause[]{
+                new StringLiteral("Enter word1 "),
```

Comment:
remove extra space here and in second basic stringliteral because the extra space exists in the optional